GOPKG_TOML_EXAMPLE.md

```diff
@@ -0,0 +1,60 @@
+# Gopkg.toml example
```

Comment:
This is more of a documentation page with examples. Let's remove "example" from the title.